packages/p2p/src/components/recommend-user/recommend-user.tsx

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Button, Icon, Text } from '@deriv/components';
 import { Localize } from 'Components/i18next';
 
+type TRecommendUserProps = {
+    is_buy_order_for_user: boolean;
+    is_user_recommended_previously: number | null;
```

Comment:
```suggestion
    is_user_recommended_previously?: number;
```

Reply:
I had to leave in null here as when the user has not be recommended before, the default value returned from backend is null

Reply:
LGTM! 👍